packages/features/settings/BookerLayoutSelector.tsx

```diff
@@ -162,7 +172,8 @@ const BookerLayoutFields = ({ settings, onChange, showUserSettings }: BookerLayo
           onValueChange={(layout: BookerLayouts) => onDefaultLayoutChange(layout)}>
           {bookerLayoutOptions.map((layout) => (
             <RadioGroup.Item
-              className="aria-checked:bg-emphasis hover:bg-subtle focus:bg-subtle w-full rounded-[4px] p-1 text-sm transition-colors"
+              className="aria-checked:bg-emphasis hover:[&:not(:disabled)]:bg-subtle focus:[&:not(:disabled)]:bg-subtle w-full rounded-[4px] p-1 text-sm transition-colors disabled:cursor-not-allowed disabled:opacity-40"
+              disabled={toggleValues[layout] === false}
```

Comment:
This part disables this layout for being a default layout, if the layout itself is not enabled.